server/endpoints/api/workspaceThread/index.js

```diff
@@ -0,0 +1,594 @@
+const { v4: uuidv4 } = require("uuid");
```

Comment:
While some of these endpoints make sense to be within a `Workspace Thread` model/object/tag. Additionally, How does this API work when not in multi-user mode as well as in multi-user?

In single-user you typically are working as the user when in the API and in multi-user you are not even a user. I see one endpoint has a param for a userID, but there is no User API - so how would one retrieve the id for such a call?

Reply:
Think we may need to expand some more API calls to user management as well as right now it would be cumbersome to find/guess user ids as i dont think we show them anywhere